src/main/java/net/wasdev/wlp/common/plugins/util/ServerFeatureUtil.java

```diff
@@ -0,0 +1,314 @@
+package net.wasdev.wlp.common.plugins.util;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.apache.commons.io.FileUtils;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import net.wasdev.wlp.common.plugins.config.XmlDocument;
+
+/**
+ * Utility class to determine server features
+ */
+public abstract class ServerFeatureUtil {
```

Comment:
If we were to extract the common logger methods as I previously suggested and move them into their own class, could we make the rest of the methods in this class static? It doesn't seem like there is any state being persisted at the class level. This would make the invocations easier in Maven; for example, you could just do `ServerFeatureUtil.getServerFeatures(serverDirectory)`.

Reply:
Addressed above, extracting the common logger methods this will be tracked as a new issue: #51.  Once that issue is addressed we could then make the rest of the methods static.